### PR TITLE
[cosmetic] TableSelector use <i> instead of <Button> for refresh

### PR DESCRIPTION
**Pull request by mistercrunch**
### Before

screen shot 2019-01-29 at 11 16 21 pm

### After

screen shot 2019-01-29 at 11 14 13 pm
screen shot 2019-01-29 at 11 14 20 pm

### CODE REVIEW

File: superset/migrations/versions/8d49a37823bf_make_table_unique_within_db_and_schema.py [hunk 1]
```diff
@@ -1,3 +1,19 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```
Comment (xtinec) [inline superset/migrations/versions/8d49a37823bf_make_table_unique_within_db_and_schema.py]: another PR bleeding through?

File: superset/migrations/versions/a33a03f16c4a_add_extra_column_to_savedquery.py [hunk 2]
```diff
@@ -1,3 +1,19 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```
Comment (xtinec) [inline superset/migrations/versions/a33a03f16c4a_add_extra_column_to_savedquery.py]: ditto

File: superset/assets/src/components/RefreshLabel.jsx [hunk 3]
```diff
@@ -33,6 +31,8 @@ class RefreshLabel extends React.PureComponent {
     this.state = {
       hovered: false,
     };
+    this.mouseOver = this.mouseOver.bind(this);
```
Comment (xtinec) [inline superset/assets/src/components/RefreshLabel.jsx]: Shouldn't need an event handler here. Can do the color change via css `:hover` like this: https://stackoverflow.com/questions/25770590/change-color-when-hover-a-font-awesome-icon

File: superset/migrations/versions/8d49a37823bf_make_table_unique_within_db_and_schema.py (hunk 1 above)
Comment (mistercrunch) [reply superset/migrations/versions/8d49a37823bf_make_table_unique_within_db_and_schema.py]: ran `rodent apply` and figured might as well add missing headers, I could split the PR but I don't think it's worth it.

File: superset/assets/src/components/RefreshLabel.jsx (hunk 3 above)
Comment (mistercrunch) [reply superset/assets/src/components/RefreshLabel.jsx]: problem is we need to apply `text-primary` and don't want to hard code the color. I guess I could use a `.less` file that would import the main less file

File: superset/migrations/versions/a33a03f16c4a_add_extra_column_to_savedquery.py (hunk 2 above)
Comment (mistercrunch) [reply superset/migrations/versions/a33a03f16c4a_add_extra_column_to_savedquery.py]: would rather not split the PR, not sure why CI allowed these files to slip in without licenses, I'm guessing they come from branching before the license check

**Comment by mistercrunch** [comment]
@xtinec thank you for the review. I addressed your comments!

File: superset/migrations/versions/a33a03f16c4a_add_extra_column_to_savedquery.py (hunk 2 above)
Comment (xtinec) [reply superset/migrations/versions/a33a03f16c4a_add_extra_column_to_savedquery.py]: It shouldn't be too hard to just unstage this file, right? :) 
Say if we cut the next apache release after this commit, it wouldn't be right to have this migration there as it is unrelated to the change (and unrelated to the release).

**Review by xtinec — CHANGES_REQUESTED** [review]
Looks good to go after the accidental migration gets unstaged.

File: superset/migrations/versions/8d49a37823bf_make_table_unique_within_db_and_schema.py (hunk 1 above)
Comment (xtinec) [reply superset/migrations/versions/8d49a37823bf_make_table_unique_within_db_and_schema.py]: It is if we cut the next build right after this PR. :)

**Comment by mistercrunch** [comment]
I'm not sure whether it mattered, but I moved the license header to this other PR https://github.com/apache/incubator-superset/pull/6801

**Pull request merged**